ios-patches/003_detect_metalkit.diff

```diff
@@ -1,20 +0,0 @@
-diff --git a/sdk/objc/Framework/Classes/Metal/RTCMTLVideoView.m b/sdk/objc/Framework/Classes/Metal/RTCMTLVideoView.m
-index e36cb3ab4..012c2ecd8 100644
---- a/sdk/objc/Framework/Classes/Metal/RTCMTLVideoView.m
-+++ b/sdk/objc/Framework/Classes/Metal/RTCMTLVideoView.m
-@@ -60,7 +60,14 @@
- 
- + (BOOL)isMetalAvailable {
```

Comment:
This change didn't go away, it now lives in the (renamed) patch `003_support_ios8.diff` and includes the DEPLOYMENT_TARGET changes which are iOS8 also requires.